src/main/java/me/arminb/sara/entities/Question.java

```diff
@@ -0,0 +1,88 @@
+package me.arminb.sara.entities;
+
+
+import java.util.Date;
+import java.util.List;
+
+public class Question {
+
+    public Date getDate() {
+        return date;
+    }
+
+    public void setDate(Date date) {
+        this.date = date;
+    }
+
+    private Date date;
```

Comment:
put createdAt, modifiedAt and id in a BaseEntity and make sure all of the entities extend from it. We are interested in knowing this information about every entity in the system.